Userland/Libraries/LibCrypto/BigInt/SignedBigInteger.h

```diff
@@ -60,6 +61,11 @@ class SignedBigInteger {
         }
         return SignedBigInteger { UnsignedBigInteger::create_from(unsigned_value), sign };
     }
+    static SignedBigInteger create_from_double(double value)
+    {
+        return create_from_double_with_multiplier(value, 1);
```

Comment:
multiplication by 1 is currently O(N) can we avoid this somehow? (and/or add a fast path to UnsignedBigInteger::multiplied_by & divided_by)